docs/conf.py

```diff
@@ -10,7 +10,8 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys, os, glob
+import os
```

Comment:
where are these changes coming from?  I think these are unrelated.  Can you either remove, or make a separate PR?

Reply:
I think they resulted from a `ruff format` run; it should be fixed now